Kubernetes/windows/debug/collectlogs.ps1

```diff
@@ -2,6 +2,12 @@ Param(
     [parameter(Mandatory = $false)] [string] $Network = "L2Bridge"
 )
 
+if (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator"))
```

Comment:
I validated that this works. But would it perhaps be simpler if we replace the 4 lines by writing the following at the top of each of the scripts:
```
#Requires -RunAsAdministrator
```
Then we can rely on PowerShell's built-in error message.

Reply:
Great suggestion, thank you!